crates/types/src/traits/block_contents.rs

```diff
@@ -145,6 +146,15 @@ pub fn precompute_vid_commitment(
 /// do dispersal for the genesis block. For simplicity and performance, we use 1.
 pub const GENESIS_VID_NUM_STORAGE_NODES: usize = 1;
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+/// Information about builder fee for proposed block
+pub struct FeeData<TYPES: NodeType> {
```

Comment:
Could we perhaps rename this? It's not so much Fee data as it is just the builder fee for a proposed block in my mind at least. Open to discussion, but, perhaps something like `BuilderFee` would be more descriptive?